doc/iterations/2023.6/shapes/shape Migration flow.adoc

```diff
@@ -0,0 +1,29 @@
+= (appetite) Shape
+
+== Problem
+
+In order to include a test which covers the all odesign to view migration path, we will migrate the existing flow model.
+
+This view model should be generated using the soon to be introduced Java APIs to create view models. https://github.com/eclipse-sirius/sirius-components/issues/1919
+
+== Key Result
+
+The flow odesign model should be converted to a view model. All concepts used in the flow model should have a respective view concept.
+
+== Solution
+
+Use the soon to be introduced Java APIs to create view models. https://github.com/eclipse-sirius/sirius-components/issues/1919
+
+=== Scenario
+
+Giving a flow model defined in a odesign file, we should be able to generate a view model which  match all the concepts existing in the odesign flow model
```

Comment:
The end user will not do that, the end user will open Sirius Web and not see and single change at all. Everything will work as before. including our Cypress integration tests which are heavily relying on flow.